src/test/java/io/getunleash/util/UnleashConfigTest.java

```diff
@@ -114,7 +113,7 @@ public void should_set_sdk_version() {
         UnleashConfig config =
                 UnleashConfig.builder().appName("my-app").unleashAPI("http://unleash.org").build();
 
-        assertThat(config.getSdkVersion()).isEqualTo("unleash-client-java:development");
```

Comment:
I don't think we're using it anywhere so should be safe to rename it to a common convention with other SDKs. The convention is inspired by npm versions since Node is a dominant part of our stack.